tsconfig.json

```diff
@@ -25,5 +25,13 @@
     "ignoreDeprecations": "5.0",
     "useDefineForClassFields": true,
     "useUnknownInCatchVariables": true
+  },
+  // For Cypress to work with `"moduleResoolution": "bundler"`
+  // https://github.com/cypress-io/cypress/issues/27731
+  "ts-node": {
+    "compilerOptions": {
+      "module": "ESNext",
+      "moduleResolution": "Node"
+    }
```

Comment:
You could also have a separate TS config for Cypress.

Reply:
We already do but the issue cannot be fixed in there (apart perhaps by duplicating the root `tsconfig` to avoid extending, but I'm not even 100% sure it would work either).